scripts/apps/authoring/authoring/directives/sd-static-autocomplete.ts

```diff
@@ -0,0 +1,125 @@
+import {debounce} from 'lodash';
+import {KEYS} from 'core/keyboard/keyboard';
+
+interface IScope extends ng.IScope {
+    value: string;
+    available: Array<string>;
+    suggestions: Array<string>;
+    activeSuggestion?: number;
+    select: (suggestion: string) => void;
+    onSelect: ({suggestion}: {suggestion: string}) => void;
+    suggested?: string;
+}
+
+export const sdStaticAutocompleteDirective = () => ({
```

Comment:
Why not add text autocomplete field to UI framework instead?

Reply:
I'm not sure actually where this should be done, but it should eventually work for react components. just wanted to make it work with current slugline field which has some integration with tansa spellcheck